### DISCUSSION TITLE
Backup e ripristino

### DISCUSSION THREAD

**Discussion by millenniumy2k** [indomus/forum#1232] [HUB: Home Assistant]
Ciao a tutti, come detto negli altri post, sono alle prime armi e sto sperimentando, leggende tutte le vostre guide...  abbiate pazienza 😅...
Ho iniziato con una installazione su windows 10 con docker... 
Vorrei sapere 2 cose:
1- come posso fare un backup ed eventualmente ripristinare tutto su altro PC?
2- prima o poi vorrei passare a un raspberry: devo rifare tutto o c'è qualche restore che  posso fare?

**Comment by snakuzzo** [comment]
Io innanzi tutto ti consiglierei l'utilizzo di docker-compose.
Detto ciò ti basta fare il backup delle directory/file montati come volumi del container. Quando dovrai tirare tutto su sul nuovo server (salvo eventuali adeguamenti di immagini ad hoc per arm/intel) ti basterà ricreare il container mappando le directory che hai backupato
